packages/dso-toolkit/src/components/logo/logo.stories-of.ts

```diff
@@ -0,0 +1,60 @@
+import { Addon_PartialStoryFn } from "@storybook/types";
+
+import { StoriesOfArguments, storiesOfFactory } from "../../storybook/index.js";
+
+import { LogoArgs, logoArgsMapper, logoArgTypes } from "./logo.args.js";
+import { Logo } from "./logo.models.js";
+
+export interface LogoTemplates<TemplateFnReturnType> {
+  logoTemplate: (logoProperties: Logo) => TemplateFnReturnType;
+}
+
+export interface LogoParameters<TemplateFnReturnType> {
+  decorator: (story: Addon_PartialStoryFn<TemplateFnReturnType>) => TemplateFnReturnType;
+}
```

Comment:
Ik denk dat jij een kopietje van het Label component hebt gemaakt 😁 

Sommige componenten willen we wrappen in een voorbeeld markup structuur. Dat doen we met decorators. Omdat de Story definitie op een plek staat die niet weet welke templating language er wordt gebruikt (lit, JSX, angular) moet de decorator als parameter worden meegegeven.

Dat dwingen we af door een property `decorator`. Maar je maakt hier helemaal geen gebruik van. De interface `LogoParameters` mag dus weg.